### ISSUE TITLE
fix a bug of class concat_text_O

### ISSUE THREAD

**Issue by yslion** [omar92/ComfyUI-QualityOfLifeSuit_Omar92#40]
return separator.join(args[:-1]) + args[-1] ----> return (separator.join(args[:-1]) + args[-1],)

**Comment by pegasKing2023** [comment]
> return separator.join(args[:-1]) + args[-1] ----> return (separator.join(args[:-1]) + args[-1],)

yes ,i have the same problem, when i use the concat_text_o node ,only  output the first char of string, such as concat "abc"+"de",only output  "a".
I use your code solve this problem

**Comment by lhkjacky** [comment]
The code `return (separator.join(args[:-1]) + args[-1],) ` is that it joins all elements except the last one with the separator (-), and then concatenates the last element without a separator. This results in a-b-cd instead of the desired a-b-c-d.

To join all elements in args with the separator. 
`return (separator.join(args),)`
